### DISCUSSION TITLE
Regex support for User highlights

### DISCUSSION THREAD

**Discussion by LittleHam** [Chatterino/chatterino2#5360] [Ideas]
### Checklist

- [X] I'm reporting a problem with Chatterino
- [X] I've verified that I'm running **the most** recent nightly build or stable release
- [X] I've looked for my problem on the [wiki](https://wiki.chatterino.com/Help/)
- [X] I've searched the [issues and pull requests](https://github.com/Chatterino/chatterino2/issues?q=) for similar looking reports

### Describe your issue

It is not currently possible to enable regex in the Users Highlights field.

Being able to use regex on usernames would help moderation efforts during bot raids or ban evasion where all names use a similar pattern.

### Screenshots

![user](https://github.com/Chatterino/chatterino2/assets/87534059/ebe08421-2138-468d-8e07-b40a7df5489a)

### OS and Chatterino Version

Chatterino Nightly 2.5.0 (commit 1caf7ca4, 2024-04-22) on Windows 10 Version 22H2

**Comment by Infinitay** [comment]
_Not an answer to your c2 request, but another possible solution_ 

> help moderation efforts during bot raids or ban evasion where all names use a similar pattern

FWIW You should consider using Fossabot's nuke feature if you aren't already. They offer the ability to timeout/nuke users **who send a particular message or that matches a regex** and also lets you ban prior and future messages for a certain duration.

https://docs.fossabot.com/nukes/usage
